ocs_ci/cleanup/aws/defaults.py

```diff
@@ -2,7 +2,7 @@
 Defaults module for AWS cleanup
 """
 AWS_REGION = 'us-east-2'
-CLUSTER_PREFIXES_TO_EXCLUDE_FROM_DELETION = {'jnk': '36'}
-MINIMUM_CLUSTER_RUNNING_TIME_FOR_DELETION = 10
+CLUSTER_PREFIXES_SPECIAL_RULES = {'jnk': 36, 'dnd': 36}
```

Comment:
The prefix `dnd` suggest *do not delete*... so *36* hours is quite short time IMHO. What about something like the `never` value mentioned above?

Reply:
Done!